draft-ietf-quic-applicability.md

```diff
@@ -487,6 +487,30 @@ Padding can also be used by an application to reduce leakage of
 information about the data that is sent. A QUIC implementation can expose an
 interface that allows an application layer to specify how to apply padding.
 
+# Error Handling
+
+QUIC recommends that endpoints that detect errors should signal the occurance to
```

Comment:
```suggestion
QUIC recommends that endpoints that detect errors signal the occurance to
```

Reply:
Or perhaps 
```suggestion
QUIC recommends that endpoints signal any detected errors to
```
...as both more concise and removes the mis-spelling of "occurrence."

Reply:
I like Mike's